test/display.test.ts

```diff
@@ -122,4 +122,21 @@ describe('display', () => {
     );
     expect(stripAnsi(actual)).toEqual(stripAnsi(expected));
   });
+
+  it('should show test path in output if path present', async () => {
+    const { scanResults } = await scan({ path: helloWorldPath });
+    const errors: string[] = [];
+    const options = { path: '/path/to/project' };
+    const actual = await display(
+      scanResults,
+      withDepOneIssueAndFix,
+      errors,
+      options,
+    );
+    const expected = await readFixture(
+      'hello-world-display',
+      'display-testing-file-path.txt',
+    );
+    expect(stripAnsi(actual)).toEqual(stripAnsi(expected));
```

Comment:
I would recommend us to either remove the ansi/terminal colors from display-testing-file-path.txt, and by doing that being able to remove the first `stripAnsi` located in `expected`, but keep the strip in `toEqual`,

or

keep the ansi in the txt-file, but remove both stripAnsi-calls, since then it should be possible to compare them.

Reply:
I've tried both options but both times the test fails, still complaining about differences in ANSI; it seems stripAnsi doesn't actually remove everything, as per the image below (I've also seen it used to match against a snapshot, so maybe that's its intended purpose). I think for now it's probably not worth spending ages researching and debugging this, as the dual use of stripAnsi follows the format of the entire test suite.
<img width="564" alt="Screenshot 2021-10-05 at 16 51 00" src="https://user-images.githubusercontent.com/62237867/136058035-122a61d6-d8e3-435b-99fb-c7f7def64ba0.png">